tests/util/test_yaml.py

```diff
@@ -372,6 +386,17 @@ def test_secrets_keyring(self):
         _yaml = load_yaml(self._yaml_path, yaml_str)
         self.assertEqual({'http': {'api_password': 'yeah'}}, _yaml)
 
+    def test_secrets_credstash(self):
+        """Test credstash fallback & get_password."""
+        yaml.credstash = None
```

Comment:
Good point, I should've known better. I was just copying the test case directly above to keep in line with existing tests/formatting